next.config.js

```diff
@@ -15,6 +15,10 @@ const nextConfig = {
   //   ],
   // },
 
+  images: {
+    unoptimized: true,
```

Comment:
why this line?

Reply:
Somehow when not having that, I got an error when trying to render the png.

Reply:
I should probably look into that again, if the unoptimized: true is really necessary

Reply:
Going to that as part of an extra task.